proc_DOSY_CPMG.py

```diff
@@ -74,10 +74,7 @@
     s.setaxis('t2', lambda x: x-echo_center)
     fl.next('request 5 (after subtracting best shift)')
     s.register_axis({'t2':0})
-    #s /= zeroth_order_ph(s['t2',0], fl=fl)
-    ph0 = s['t2',0].C.mean('t2')
-    ph0 /= abs(ph0)
-    s /= ph0
+    s /= zeroth_order_ph(s['t2':0])
     fl.image(s)
     time_bound = min(abs(s.getaxis('t2')[r_[0,-1]]))
     s = s['t2':(-time_bound,time_bound)]
```

Comment:
this section of the code here and surrounding is what will solve #20 (note for @guinneak198 ) -- please also see the other mentions of #20 : specifically where I talk about extracting this as a second order function

Reply:
@guinneak198 : I think that what will solve #20 is for you to make this into a second order function (it uses zeroth order), so that you can use it in other places as well --> something like center_CPMG_echo()